tests/smoke_test.py

```diff
@@ -0,0 +1,68 @@
+"""Smoke test for the autogpt package."""
```

Comment:
Looks like this may have been removed in the past couple of days?  Lmk if we want to decorate this as requiring an API key or delete it.